StRoot/StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -854,10 +854,11 @@ Int_t StPicoDstMaker::MakeWrite() {
 
   mBField = muEvent->magneticField();
 
+#if !defined (__TFG__VERSION__)
```

Comment:
`StRoot/StPicoDst*` is of course your responsibility... But you are making not just your life harder by introducing these preprocessor branchings.

Reply:
I think this is an incorrect statement.